jumpscripts/healthchecks/vm_disk_ratio.py

```diff
@@ -30,7 +30,7 @@ def action(gid=None):
 
     vmachines_count = cbcl.vmachine.count({"status": {"$nin": ["ERROR", "DESTROYED"]}})
     disks_count = len(ovscl.get("/vdisks", params={})["data"])
-    if vmachines_count > 0:
+    if vmachines_count > 0 and disks_count > 100:
```

Comment:
Why 100? Shouldn't that depend on the number of machines

Reply:
100 because when there are only few vms / disks on the env, due to rounding problems we can get false positives on this error